ui/raidboss/data/02-arr/trial/titan-nm.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.TheNavel,
+  timelineFile: 'titan-nm.txt',
+  triggers: [
+    {
+      id: 'TitanNm Tumult',
+      netRegex: NetRegexes.ability({ id: '282', source: 'Titan', capture: false }),
+      condition: (data) => data.role === 'healer' || data.CanAddle(),
+      suppressSeconds: 2,
+      response: Responses.aoe(),
+    },
+    {
+      // Gaol callout for both yourself and others
+      id: 'TitanNm Gaols',
+      netRegex: NetRegexes.gainsEffect({ effectId: '124' }),
+      condition: (data) => data.role === 'healer',

```

Comment:
Why are only healers being notified for Gaols?

Reply:
I actually didnt notice that, I will replace it. I copied it from another shorthand condition and forget to change it. I'll need to change it to the way I have the Esuna trigger in Brayflox. if then statement.